src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/server/commands/ArsMagicaLegacyCommand.java

```diff
@@ -0,0 +1,691 @@
+                                                .executes(ArsMagicaLegacyCommand::addSkillPointSelf))))
+                        .then(Commands.literal("consume")
+                                .then(Commands.argument("target", EntityArgument.players())
+                                        .then(Commands.argument("skill_point", ResourceLocationArgument.id())
+                                                .suggests(SUGGEST_SKILL_POINTS)
+                                                .executes(ArsMagicaLegacyCommand::consumeOneSkillPoint)
+                                                .then(Commands.argument("amount", IntegerArgumentType.integer(0))
+                                                        .executes(ArsMagicaLegacyCommand::consumeSkillPoint))))
+                                .then(Commands.argument("skill_point", ResourceLocationArgument.id())
+                                        .suggests(SUGGEST_SKILL_POINTS)
+                                        .executes(ArsMagicaLegacyCommand::consumeOneSkillPointSelf)
+                                        .then(Commands.argument("amount", IntegerArgumentType.integer(0))
+                                                .executes(ArsMagicaLegacyCommand::consumeSkillPointSelf))))
+                        .then(Commands.literal("set")
+                                .then(Commands.argument("target", EntityArgument.players())
+                                        .then(Commands.argument("skill_point", ResourceLocationArgument.id())
+                                                .suggests(SUGGEST_SKILL_POINTS)
+                                                .then(Commands.argument("amount", IntegerArgumentType.integer(0))
+                                                        .executes(ArsMagicaLegacyCommand::setSkillPoint))))
+                                .then(Commands.argument("skill_point", ResourceLocationArgument.id())
+                                        .suggests(SUGGEST_SKILL_POINTS)
+                                        .then(Commands.argument("amount", IntegerArgumentType.integer(0))
+                                                .executes(ArsMagicaLegacyCommand::setSkillPointSelf))))
+                        .then(Commands.literal("get")
+                                .then(Commands.argument("target", EntityArgument.players())
+                                        .then(Commands.argument("skill_point", ResourceLocationArgument.id())
+                                                .suggests(SUGGEST_SKILL_POINTS)
+                                                .executes(ArsMagicaLegacyCommand::getSkillPoint)))
+                                .then(Commands.argument("skill_point", ResourceLocationArgument.id())
+                                        .suggests(SUGGEST_SKILL_POINTS)
+                                        .executes(ArsMagicaLegacyCommand::getSkillPointSelf))))
+        );
+    }
```

Comment:
Move each command to its own class like Skill command was before

Reply:
I can move them to their own class, yes, but I'd still keep them as one command (that's how similar mods, like Astral Sorcery, Minecolonies etc. do it as well)